manager/controllers/step_controller.go

```diff
@@ -62,6 +62,10 @@ type hash struct {
 var clusterName = os.Getenv(dfv1.EnvClusterName)
 
 func init() {
+	if clusterName == "" {
```

Comment:
I don't want to do this. If two clusters use the name name, there will be incorrect behaviour. It is correct for the quick-start to have "default" as the name.